### PR TITLE
ST-1836 implement new header design

### PR DESCRIPTION
**Pull request by goldenmaya**
Description:
https://virtocommerce.atlassian.net/browse/ST-1836

--

QA-test: ST-1839

Demo-test:

Download artifact URL: https://vc3prerelease.blob.core.windows.net/packages/vc-theme-b2b-vue-1.3.0-pr-168-b5b8d7e2.zip

### CODE REVIEW

**Review by Flanker72 — APPROVED** [review]

File: client-app/shared/layout/components/header/_internal/bottom-header.vue [hunk 1]
```diff
@@ -54,43 +61,71 @@
...
 import { useCompareProducts } from "@/shared/compare";
 import { useUser } from "@/shared/account";
+const { t } = useI18n();
 
 const { organization } = useUser();
 const { cart } = useCart();
-const { mainMenuLinks } = useNavigations();
+const { desktopCatalog } = useNavigations();
 const { productsIds } = useCompareProducts();
 const { searchBarVisible, showSearchBar } = useSearchBar();
```
Comment (Flanker72) [inline client-app/shared/layout/components/header/_internal/bottom-header.vue:85]: This row and below one are not needed anymore, please check unused imports/variables.

File: client-app/shared/layout/components/search-bar/search-bar.vue [hunk 2]
```diff
@@ -1,99 +1,86 @@
 <template>
-  <div
-    class="flex items-center justify-between px-12 py-7 bg-[color:var(--color-search-bar-bg)] select-none"
-    v-click-outside="() => searchDropdownVisible && hideSearchDropdown()"
-  >
-    <router-link to="/">
-      <VcImage :src="$cfg.logo_inverted_image" class="h-12 mr-8" />
```
Comment (Flanker72) [inline client-app/shared/layout/components/search-bar/search-bar.vue:7]: `VcImage` isn't used anywhere in the file, however it still present in imports, causing warning. Please, remove VcImage import from script tag.

File: client-app/shared/layout/composables/useNavigations.ts [hunk 3]
```diff
@@ -27,6 +28,22 @@ const mainMenuLinks = computed<MenuLink[]>(() =>
...
 );
 
+const desktopCatalog = computed<MenuLink>(
+  () =>
+    ({
+      id: "all-products-menu",
+      route: {
+        name: "Catalog",
+      },
+      title: globals.i18n!.global.t("shared.layout.header.catalog"),
```
[the remark this answers is not in the document]
Comment (artmakarov) [reply client-app/shared/layout/composables/useNavigations.ts:38]: > Please use useI18n approach t = useI18n(); t("shared.layout.header.catalog")

`useI18n`  in this case it does not work

[the remark this answers is not in the document]
Comment (artmakarov) [reply client-app/shared/layout/composables/useNavigations.ts:38]: > Please use useI18n approach t = useI18n(); t("shared.layout.header.catalog")

`useI18n` in this case it does not work

**Review by Flanker72 — APPROVED** [review]

**Review by trueboroda — APPROVED** [review]

**Pull request merged**